src/components/GuestCheckout.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="max-w-lg m-auto my-8 py-20 px-10">
+    <h3 class="text-2xl mb-6 text-gray-700 tracking-widest">Guest Checkout</h3>
+    <p class="pb-10 text-gray-700 pt-2">
```

Comment:
The idea with this here is to delineate the difference between a guest and a user. Guest's should not have access to dashboards